src/EasyProfiler.MariaDb/Context/ProfilerDbContext.cs

```diff
@@ -39,7 +40,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
                 entity
                     .Property(p => p.Duration)
-                    .HasColumnType("time");
+                    .HasColumnType("bigint")
+                    .HasConversion(new TimeSpanToTicksConverter());
```

Comment:
Change columnt type 'time' to 'bigint' and add conversion method. 
fyi @selcukkutuk

Reply:
I had not checked this situation.  Thanks.👍🏻